spec/frame_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'frame'
+
+describe Frame do
+
```

Comment:
Maybe start separating out your tests by grouping tests under another describe for each method. For example here have describe '#total' then your it statement 'returns sum of 2 rolls'.